examples/fluent-bit/grafana-cloud/README.md

```diff
@@ -0,0 +1,24 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit loki Plugin Documentation](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki). Be aware there is a separate Golang output plugin provided by [Grafana](https://grafana.com/docs/loki/latest/clients/fluentbit/) with different configuration options. Here is a [blog](https://calyptia.com/blog/how-to-send-logs-to-loki-using-fluent-bit) on sending logs to Loki using the loki Fluent Bit plugin. The Loki data source provides access to Loki, Grafana’s log aggregation system.
+
+It should be noted that this example and Grafana docs show `"enable-ecs-log-metadata":"true"` (which is the default). This option tells FireLens to add ECS Task Metadata keys to logs.  
+
+For all configuration parameters for Fluent Bit Loki output plugin, see [Fluent Bit loki Plugin Documentation](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki) documentation. If you are looking for support for bearer_token , please follow the [Fluent Bit loki latest Documentation](https://docs.fluentbit.io/manual/pipeline/outputs/loki)
```

Comment:
> If you are looking for support for bearer_token , please follow the [Fluent Bit loki latest Documentation](https://docs.fluentbit.io/manual/pipeline/outputs/loki)

Thanks for adding this. But this sentence does not end with a period. Also, I think we need to tell them to use the docker hub upstream image with this: https://hub.docker.com/r/fluent/fluent-bit

Reply:
Done